src/sentry/static/sentry/app/views/settings/organizationTeams/teamDetails.jsx

```diff
@@ -62,15 +61,62 @@ const TeamDetails = createReactClass({
     });
   },
 
+  handleRequestAccess() {
```

Comment:
This is added here so that if user accesses a team details page (e.g. linked by someone) they can request access.